napari/layers/shapes/shapes.py

```diff
@@ -1583,7 +1595,7 @@ def _view_text_coords(self) -> Tuple[np.ndarray, str, str]:
         ]
 
         return self.text.compute_text_coords(
-            sliced_in_view_coords, ndisplay, order
+            np.asarray(sliced_in_view_coords), ndisplay, order
         )
```

Comment:
`compute_text_coords`,  `calls get_text_anchors`, which dependings on it's parameters calls `_calculate_anchor_center`, itself using `_calculate_bbox_centers`. 

_calculate_bbox_centers behavior depends on wether it is given a  list or an ndarray, and create a failure. 

as `_calculate_bbox_centers` seem to be only used for this specific purpose I'm tempted to assume this is a bug in `_calculate_bbox_centers` which behavior should be fixed, likely converted from list to ndarray or vice-versa higher up in the stack.

My take is to mark this specific one right here with a temporary `# type: ignore`. and have a gh issue for it